server/api/crud/model_monitoring/deployment.py

```diff
@@ -1086,11 +1086,22 @@ def set_credentials(
                                              for example taosws://<username>:<password>@<host>:<port>.
         :param _default_secrets_v3io:     Optional parameter for the upgrade process in which the v3io default secret
                                           key is set.
+        :raise MLRunConflictError:        If the credentials are already set for the project and the user
+                                          provided new creds.
+        :raise MLRunInvalidMMStoreType:   If the user provided invalid credentials.
         """
         try:
             self.check_if_credentials_are_set(only_project_secrets=True)
+            if self._is_the_same_cred(
+                endpoint_store_connection, stream_path, tsdb_connection
+            ):
+                logger.warning(
+                    "The same credentials are already set for the project",
+                    project=self.project,
+                )
```

Comment:
Why use `warning` severity?
If anything, I would use `info` severity - but do you think we need any log?

Reply:
I agree the severity is wrong, maybe I can change it to debug because mostly it is for debugging.. I put it there because I think we need to understand somehow that the api didn't really changed anything.

Reply:
`debug` is good as well (assuming that's the default severity). Maybe just to clarify change too 
```suggestion
                logger.debug(
                    "The same credentials are already set for the project - aborting with no error",
                    project=self.project,
                )
```